### DISCUSSION TITLE
How to disable sbt version update

### DISCUSSION THREAD

**Discussion by ngu04** [scala-steward-org/scala-steward#3001] [Q&A] [answered]
I guess //scala-steward:off will work only for sbt dependencies . 

I want restriction for sbt update .   Don't see any example in `.scala-steward.conf` for disabling sbt version update

**Comment by exoego** [comment, marked as answer]
```
updates.ignore = [ { groupId = "org.scala-sbt", artifactId = "sbt" } ]
```

PR description by Scala Steward includes such example.
